src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskProvidingConfiguration.swift

```diff
@@ -56,7 +56,12 @@ extension RiskProvidingConfiguration {
 		return currentDate < exposureDetectionValidUntil(lastExposureDetectionDate: lastExposureDetectionDate)
 	}
 
-	func shouldPerformExposureDetection(lastExposureDetectionDate: Date?, currentDate: Date = Date()) -> Bool {
+	func shouldPerformExposureDetection(activeTracingHours: Int, lastExposureDetectionDate: Date?, currentDate: Date = Date()) -> Bool {
+		// Don't allow exposure detection within the first frame of exposureDetectionInterval
+		if activeTracingHours < ((exposureDetectionInterval.day ?? 1) * 24) {
```

Comment:
personal preference: use guard here, however feel free to ignore :D

Reply:
:white_check_mark: